pom.xml

```diff
@@ -343,6 +344,11 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>${junit4.version}</version>
+            </dependency>
```

Comment:
This JUnit4 dependency needs to be added because we need to check the `AssumptionViolationException` that is returned by the `assumeThat` method from `assertj` package. 
I guess using Assumptions from [JUnit5](https://junit.org/junit5/docs/snapshot/user-guide/#writing-tests-assumptions) would avoid using any JUnit4 package.

Reply:
> I guess using Assumptions from [JUnit5](https://junit.org/junit5/docs/snapshot/user-guide/#writing-tests-assumptions) would avoid using any JUnit4 package.

Yes, let's try that.   We don't want the direct dependency on junit4

Reply:
Actually, having poked at this myself, it is nightmare.  If you go the Junit5 route, you then need to make the opentest dependency compile time.  I'd go back to the stringifying the classname (as you originally had)

Reply:
Yeah, my intention was that, to avoid using more dependencies only for that simple check.